doc/guides/contributing/pull-requests.md

```diff
@@ -121,7 +121,7 @@ If you are modifying code, please be sure to run `make lint` (or
 code style guide.
 
 Any documentation you write (including code comments and API documentation)
-should follow the [Style Guide](../doc-style-guide.md). Code samples
+should follow the [Style Guide](../../README.md). Code samples
```

Comment:
/doc/ is missing in the link

Reply:
> /doc/ is missing in the link

That would result in an incorrect path, wouldn't it?

Previously, the target was `/doc/guides/doc-style-guide.md`.  Since the file here is `doc/guides/contributing/pull-requests.md`, it used the relative path `../doc-style-guide.md` and it was fine.

So, `../../README.md` should end up being `/doc/README.md`. And even if it didn't, `README.md` in the project route isn't a broken link anyway.

Reply:
(And `make lint-md` is passing for me locally.  ¯\\_(ツ)_/¯)

Reply:
Sorry, my mistake for looking at this too early in the morning 😅

Reply:
> (And `make lint-md` is passing for me locally. ¯\_(ツ)_/¯)

Oh, wait, no, it is not....

Reply:
I see. It's a different relative link that needs to be updated.

Reply:
Fixed and force-pushed.